libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFOAuthOrgAuthConfiguration.m

```diff
@@ -70,7 +70,7 @@ - (BOOL)useNativeBrowserForAuth {
 - (NSString *)loginPageUrl {
     NSString *loginPageUrl = nil;
     NSDictionary *loginPage = self.authConfigDict[kAuthConfigLoginPageKey];
-    if (loginPage) {
+    if (![loginPage isKindOfClass:[NSNull class]] && [loginPage count] > 0) {
         loginPageUrl = loginPage[kAuthConfigLoginPageUrlKey];
```

Comment:
@ssraza We should guard the retrieval of kAuthConfigLoginPageUrlKey  too.

Reply:
why we want to `kAuthConfigLoginPageUrlKey` 🤔. `loginPageUrl` is a nullable property right? will it not gonna return `nit` if value is not there?

Reply:
You are right that's fine.